README.md

```diff
@@ -93,18 +95,11 @@ $ aws cloudformation create-stack \
         ParameterKey=SNSSubscriptionEmail,ParameterValue="email-address" \
         ParameterKey=IntegrationHttpEndpoint,ParameterValue='"endpoint-url"'
 ```
-
-## 3. Configure SSL/HTTPS  
-
 The stack takes only a few minutes to deploy. It creates a new REST API in API Gateway with two stages: “PreProd” and “Prod”. A stage defines the path through which an API deployment is accessible. Each stage is deployed with its own client-side certificate. 
 
 When the deployment has completed successfully, you’ll receive an email to confirm a subscription to the Amazon SNS topic created by the template. Choose the *Confirm subscription* link in the message to subscribe to emails that are sent whenever there is an expiring certificate. A Lambda function checks once a day to see if the client certificate is expiring in 7, 3, or 1 days.
 
-In addition to confirming your subscription to certificate expiration notices, you'll need to configure the SSL/HTTPS connection between the API Gateway and your backend system. 
```

Comment:
is the certificate going to be auto-rotated or is there a separate notification mechanism that does not need them to accept subscription

Reply:
That text is discussing the API Gateway cert system, which I did not modify.  I believe I removed it because it seemed confusing in the context of the sample API backend.  You can disregard, or re-read for flow once decisions are made regarding any changes to the main README.md.

Reply:
As for LetsEncrypt rotation, that is mentioned in the last line of ./sample-api-server/README.md  
  
*To auto-renew the Let's Encrypt certificate, run the following as a scheduled command on your host. docker exec api /usr/bin/certbot renew*
  
I do not document rotating the API Gateway cert, but do document how to change it under the *Host Certificate Setup* section of the document.